.travis.yml

```diff
@@ -42,7 +42,8 @@ notifications:
     - joshua.suereth@typesafe.com
 
 script:
-  - sbt -J-XX:ReservedCodeCacheSize=128m "$SCRIPTED_TEST"
+  - sbt ";publishLocal;project bundledLauncherProj;publishLocal;project precompiled-2_9_2;publishLocal;project precompiled-2_8_2;publishLocal;project precompiled-2_9_3;publishLocal"     
```

Comment:
If this is required, we should fix the build script instead of putting it in `.travis.yml` so all required JARs are published locally before the scripted tests.